src/lib/migrations/routines/03-add-tools-in-settings.ts

```diff
@@ -0,0 +1,24 @@
+import type { Migration } from ".";
+import { collections } from "$lib/server/database";
+import { ObjectId } from "mongodb";
+
+const addToolsToSettings: Migration = {
+	_id: new ObjectId(3),
+	name: "Add empty 'tools' record in settings",
+	up: async () => {
+		const { settings } = collections;
+
+		// Find all assistants whose modelId is not in modelIds, and update it to use defaultModelId
+		await settings.updateMany(
+			{
+				tools: { $exists: false },
```

Comment:
we should probably create `settings` index on `tools` like this one https://github.com/huggingface/chat-ui/blob/dbc0ca76f290ad222f93da5dc7beb3966c011cfc/src/lib/server/database.ts#L188